spec/concepts/stellar_base/deposit_requests/operations/create_spec.rb

```diff
@@ -10,7 +10,15 @@ module Operations
               .and_return("MEMO")
             expect(ConfiguredClassRunner).to receive(:call)
               .with(GetMaxAmount.to_s).and_return(100)
-            expect(ConfiguredClassRunner).to receive(:call).with(GetHow.to_s)
+            expect(DetermineHow).to receive(:call)
+              .with(GetHow.to_s, {
+                asset_code: "BTCT",
+                account: "G-STELLAR-ACCOUNT",
+                account_id: "G-STELLAR-ACCOUNT",
```

Comment:
There's no `account_id` according to [this](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#deposit)?